addons/irlight/functions/fnc_initItemContextMenu.sqf

```diff
@@ -25,23 +25,26 @@
             private _baseClass = getText (configFile >> "CfgWeapons" >> _item >> "baseWeapon");
             _item != _baseClass + _variant
         }, {
-            params ["", "", "_item", "", "_variant"];
+            params ["_unit", "", "_item", "_slot", "_variant"];
 
             private _baseClass = getText (configFile >> "CfgWeapons" >> _item >> "baseWeapon");
 
-            ACE_player removePrimaryWeaponItem _item;
-            ACE_player addPrimaryWeaponItem (_baseClass + _variant);
-            playSound "click";
-
-            if (_turnedOn) then {
```

Comment:
doesn't matter now, but `_turnedOn` was never defined?

Reply:
Yep, that's why I removed it.